iina/FileGroup.swift

```diff
@@ -17,7 +17,6 @@ class FileInfo: Hashable {
   var filename: String
   var ext: String
   var nameInSeries: String?
-  var characters: [Character]
```

Comment:
Are you sure there's no performance issue caused by accessing `characters` and constructing `Index` in each iteration?